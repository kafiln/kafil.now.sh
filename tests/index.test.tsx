import { render } from '@testing-library/react';
import App from '../pages/index';

describe('App', () => {
  it('renders without crashing', () => {
    const { asFragment } = render(<App />);
    expect(asFragment()).toMatchSnapshot();
  });
});
