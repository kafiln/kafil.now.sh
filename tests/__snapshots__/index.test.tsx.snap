// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders without crashing 1`] = `
<DocumentFragment>
  <div
    class="css-s92abg"
  >
    <header
      class="css-1dz0hm6"
    >
      <div
        class="css-70qvj9"
        style="cursor: pointer;"
      >
        <span
          class="chakra-avatar css-1ruhx4w"
        >
          <div
            aria-label="Kafil Nasdami"
            class="chakra-avatar__initials css-0"
          >
            KN
          </div>
        </span>
        <h2
          class="chakra-heading css-h1djnr"
        >
          Kafil
        </h2>
      </div>
      <nav
        class="chakra-stack css-dr9pdj"
      >
        <ul
          class="css-1rtm76s"
        >
          <li
            class="css-1oq6uh7"
            style="list-style: none; text-transform: capitalize;"
          >
            <a
              class="chakra-link css-0"
              href="/projects"
            >
              projects
            </a>
          </li>
          <li
            class="css-1oq6uh7"
            style="list-style: none; text-transform: capitalize;"
          >
            <a
              class="chakra-link css-0"
              href="/about"
            >
              about
            </a>
          </li>
          <li
            class="css-1oq6uh7"
            style="list-style: none; text-transform: capitalize;"
          >
            <a
              class="chakra-link css-0"
              href="/blog"
            >
              blog
            </a>
          </li>
          <li
            class="css-1oq6uh7"
            style="list-style: none; text-transform: capitalize;"
          >
            <a
              class="chakra-link css-0"
              href="/contact"
            >
              contact
            </a>
          </li>
        </ul>
        <div
          class="css-1o63v0h"
        >
          <button
            class="chakra-button css-oqfhsh"
            type="button"
          >
            <svg
              fill="currentColor"
              height="1em"
              stroke="currentColor"
              stroke-width="0"
              viewBox="0 0 24 24"
              width="1em"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M3 18h18v-2H3v2zm0-5h18v-2H3v2zm0-7v2h18V6H3z"
              />
            </svg>
          </button>
        </div>
        <div
          class="css-0"
          style="cursor: pointer;"
        >
          <svg
            fill="currentColor"
            height="1.5em"
            stroke="currentColor"
            stroke-width="0"
            viewBox="0 0 512 512"
            width="1.5em"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M283.211 512c78.962 0 151.079-35.925 198.857-94.792 7.068-8.708-.639-21.43-11.562-19.35-124.203 23.654-238.262-71.576-238.262-196.954 0-72.222 38.662-138.635 101.498-174.394 9.686-5.512 7.25-20.197-3.756-22.23A258.156 258.156 0 0 0 283.211 0c-141.309 0-256 114.511-256 256 0 141.309 114.511 256 256 256z"
            />
          </svg>
        </div>
      </nav>
    </header>
    <main
      class="css-2w44fa"
    >
      <section
        class="chakra-stack css-180nsro"
      >
        <div
          style="display: inline-block; max-width: 100%; overflow: hidden; position: relative; box-sizing: border-box; margin: 0px;"
        >
          <div
            style="box-sizing: border-box; display: block; max-width: 100%;"
          >
            <img
              alt=""
              aria-hidden="true"
              role="presentation"
              src="data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iNTEyIiBoZWlnaHQ9IjUxMiIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIiB2ZXJzaW9uPSIxLjEiLz4="
              style="max-width: 100%; display: block;"
            />
          </div>
          <img
            alt="A person using a laptop"
            decoding="async"
            src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
            style="visibility: hidden; position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
          />
        </div>
        <div
          class="chakra-stack css-f3zt4v"
        >
          <div
            class="chakra-stack css-1qu80z5"
          >
            <p
              class="chakra-text css-1ezsviu"
            >
              Hello, my name is
            </p>
            <h1
              class="chakra-heading css-0"
            >
              Kafil Nasdami
            </h1>
            <h2
              class="chakra-heading css-0"
            >
              Full Stack C# Javascript developer
            </h2>
            <div
              class="chakra-container css-y8aj3r"
            >
              I love programming, which makes what I do not only a job but also a passion. As part of my previous experiences, I've build a lot of websites and web applications mainly from scratch. I code with C# but also Java I'm very comfortable around .NET technologies such as ASP.NET MVC, SQL Server, Entity framework... I also use bootstrap, javascript jquery and other Javascript technologies such as AngularJS and React ... If you ever need a reliable resource that would get the job done in a convenient period of time, then I guess I am your man.
            </div>
          </div>
          <div
            class="css-1l4w6pd"
          >
            <div
              class="chakra-button__group css-10rwr67"
              role="group"
            >
              <button
                class="chakra-button css-1iirvu6"
                type="button"
              >
                Contact
              </button>
              <button
                class="chakra-button css-1iirvu6"
                type="button"
              >
                View Resume
              </button>
            </div>
          </div>
        </div>
      </section>
    </main>
    <footer
      class="css-1dz0hm6"
    >
      <h1
        class="chakra-heading css-h1djnr"
      >
        Kafil NASDAMI
      </h1>
      <ul
        class="chakra-stack css-w11xpy"
      >
        <li
          class="css-0"
          style="list-style: none;"
        >
          <a
            aria-label="github"
            href="https://github.com/kafiln"
            rel="noopener noreferrer"
            target="_blank"
          >
            <svg
              fill="currentColor"
              height="2em"
              stroke="currentColor"
              stroke-width="0"
              viewBox="0 0 496 512"
              width="2em"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"
              />
            </svg>
          </a>
        </li>
        <li
          class="css-0"
          style="list-style: none;"
        >
          <a
            aria-label="twitter"
            href="https://twitter.com/kafiiil"
            rel="noopener noreferrer"
            target="_blank"
          >
            <svg
              fill="currentColor"
              height="2em"
              stroke="currentColor"
              stroke-width="0"
              viewBox="0 0 512 512"
              width="2em"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M459.37 151.716c.325 4.548.325 9.097.325 13.645 0 138.72-105.583 298.558-298.558 298.558-59.452 0-114.68-17.219-161.137-47.106 8.447.974 16.568 1.299 25.34 1.299 49.055 0 94.213-16.568 130.274-44.832-46.132-.975-84.792-31.188-98.112-72.772 6.498.974 12.995 1.624 19.818 1.624 9.421 0 18.843-1.3 27.614-3.573-48.081-9.747-84.143-51.98-84.143-102.985v-1.299c13.969 7.797 30.214 12.67 47.431 13.319-28.264-18.843-46.781-51.005-46.781-87.391 0-19.492 5.197-37.36 14.294-52.954 51.655 63.675 129.3 105.258 216.365 109.807-1.624-7.797-2.599-15.918-2.599-24.04 0-57.828 46.782-104.934 104.934-104.934 30.213 0 57.502 12.67 76.67 33.137 23.715-4.548 46.456-13.32 66.599-25.34-7.798 24.366-24.366 44.833-46.132 57.827 21.117-2.273 41.584-8.122 60.426-16.243-14.292 20.791-32.161 39.308-52.628 54.253z"
              />
            </svg>
          </a>
        </li>
        <li
          class="css-0"
          style="list-style: none;"
        >
          <a
            aria-label="linkedin"
            href="https://www.linkedin.com/in/kafiln/"
            rel="noopener noreferrer"
            target="_blank"
          >
            <svg
              fill="currentColor"
              height="2em"
              stroke="currentColor"
              stroke-width="0"
              viewBox="0 0 448 512"
              width="2em"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M416 32H31.9C14.3 32 0 46.5 0 64.3v383.4C0 465.5 14.3 480 31.9 480H416c17.6 0 32-14.5 32-32.3V64.3c0-17.8-14.4-32.3-32-32.3zM135.4 416H69V202.2h66.5V416zm-33.2-243c-21.3 0-38.5-17.3-38.5-38.5S80.9 96 102.2 96c21.2 0 38.5 17.3 38.5 38.5 0 21.3-17.2 38.5-38.5 38.5zm282.1 243h-66.4V312c0-24.8-.5-56.7-34.5-56.7-34.6 0-39.9 27-39.9 54.9V416h-66.4V202.2h63.7v29.2h.9c8.9-16.8 30.6-34.5 62.9-34.5 67.2 0 79.7 44.3 79.7 101.9V416z"
              />
            </svg>
          </a>
        </li>
      </ul>
    </footer>
  </div>
</DocumentFragment>
`;
